src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+
+import { Status } from '../../types/Status';
+
+type Props = {
+  status: Status,
+  onStatusChange: (status: Status) => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({ status, onStatusChange }) => {
+  return (
+    <footer className="todoapp__footer">
+      {/* Hide the footer if there are no todos */}
+      <span className="todo-count">
+        3 items left
```

Comment:
We need to calculate items in todolist

Reply:
agree with [comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/673/files#r1276513975)